### ISSUE TITLE
Connecting through TUN mode is not working on Fedora workstation (env_reset is forced in sudoers)

### ISSUE THREAD

**Issue by uncharted-user** [AdguardTeam/AdGuardVPNCLI#58]
### Please answer the following question for yourself before submitting an issue

- [X] I checked to make sure that this issue has not already been filed

### AdGuard VPN CLI version

v1.0.0 and v1.1.112

### Environment

- OS: Fedora Linux 40 (Workstation Edition)
- Kernel: 6.10.11-200.fc40.x86_64 
- Desktop: GNOME 46.5 
- Session: Wayland

### Issue Details

Steps to reproduce:
1. Open Terminal and type `curl -fsSL https://raw.githubusercontent.com/AdguardTeam/AdGuardVPNCLI/master/scripts/release/install.sh | sh -s -- -v`
2. Then login using `adguardvpn-cli login`
3. After that, connect using `adguardvpn-cli connect -l SG` → Type yes then type no and again type no

### Expected Behavior

Should show connected to SG

### Actual Behavior

Failed to start the VPN service in the background

### Screenshots

<details><summary>Screenshot 1</summary>

</details>

**Comment by sfionov** [comment]
@uncharted-user Please try `export SSL_CERT_FILE=/etc/pki/tls/cert.pem`

**Comment by uncharted-user** [comment]
> @uncharted-user Please try `export SSL_CERT_FILE=/etc/pki/tls/cert.pem`

Just tried but still getting facing same problem.

**Comment by sfionov** [comment]
@uncharted-user Can you please reproduce the problem, export logs using `adguardvpn-cli export-logs` and send them to devteam@adguard.com, mentioning this task in the message?

**Comment by uncharted-user** [comment]
> @uncharted-user Can you please reproduce the problem, export logs using `adguardvpn-cli export-logs` and send them to [devteam@adguard.com](mailto:devteam@adguard.com), mentioning this task in the message?

Thank you for the reply. Cloud you please check the logs I sent?

**Comment by sfionov** [comment]
@uncharted-user Yes, received the logs. The problem is that Fedora has `sudoers` configuration which prevents `-E` sudo option from working. We'll try to cimcumvent this in the next release.

**Comment by uncharted-user** [comment]
> We'll try to cimcumvent this in the next release.

Thank you for the consideration.

**Comment by natema** [comment]
I get the same issue on Fedora 41: 
```
❯ adguardvpn-cli connect 
[sudo] password for marvin: 
Failed to start the VPN service in the background. Process started but no pid file created
```
